src/components/search/Search.jsx

```diff
@@ -71,14 +60,23 @@ const Search = (props) => {
     )
   } else if (props.resultAuthority) {
     results = (
-      <QASearchResults key="search-results" />
+      <QASearchResults {...props} key="search-results" />
```

Comment:
I'm not a big fan of this pattern, because it's not immediately clear which parameters need to be passed.